src/config.js

```diff
@@ -46,6 +55,13 @@ const getStylesheetFilepath = () =>
 const getPageTitle = () =>
 	config.pageTitle || process.env.JEST_HTML_REPORTER_PAGE_TITLE || 'Test report';
```

Comment:
I know this isn't related to this specific PR but.
Isn't it more common to have defined env variable take precedence over config passed through package.json. 
At least that's how I tried to use it before digging in the code and finding out about the order

Reply:
Interesting!
Personally I rarely work with environment variables, and the addition of the option to use them in the plugin was community-provided - and so i haven't personally given it that much thought. But after a bit of researching it does seem that you are right in that sense that it would make more sense to go `environment variable` > `local config` > `default value`.

I will look into implementing it into this release as well